pkg/cmd/scan.go

```diff
@@ -250,6 +252,7 @@ func scanRun(opts *pkg.ScanOptions) error {
 
 	analysis, err := ctl.Run()
 	if err != nil {
+		WriteAlert(alerter)
```

Comment:
With this approach we're going to output alert multiples times since we output them in the analyzer too. As I said above, we are using alerts for another purpose for what they were originally designed, and I'm pretty sure that it will lead us to bugs. 

We should find another way to show state errors when every state failed. WDYT of logging in warning failed states in the logs. Then in the final error message, instead of saying that we should look in alerts for details, we can say that you may want to set `LOG_LEVEL=warning` to check details ?

So we can keep the alerter component as it.

It will show alerts at the end of output if at least one `--from` succeeded. 

When each `--from` failed then we can ask user to check out the warning logs to find why.

WDYT ?

Reply:
hmm nice I like it cause I think the alert behavior is good for a successfull run I was just looking for a way to give insight when all from fails. The only issue will be that you need a full second run to check what the error was :sweat: but I think this idea is ok for me ? Should I go with the change or do we need to discuss this more ?

Reply:
ok find an other way of doing it I should have done this at the begining...
![image](https://user-images.githubusercontent.com/4931174/133498719-195b8bc3-850f-45aa-806d-1488715db1ec.png)